web/permissions/permissions.go

```diff
@@ -0,0 +1,158 @@
+package permissions
+
+import (
+	"errors"
+	"fmt"
+	"net/http"
+	"strings"
+
+	"github.com/cozy/cozy-stack/pkg/consts"
+	"github.com/cozy/cozy-stack/pkg/couchdb"
+	"github.com/cozy/cozy-stack/pkg/crypto"
+	"github.com/cozy/cozy-stack/pkg/instance"
+	"github.com/cozy/cozy-stack/pkg/permissions"
+	"github.com/cozy/cozy-stack/web/middlewares"
+	"github.com/labstack/echo"
+	jwt "gopkg.in/dgrijalva/jwt-go.v3"
+)
+
+var readable = true
+var none = false
+
+var blackList = map[string]bool{
+	consts.Sessions:  none,
+	consts.Files:     readable,
```

Comment:
Yes, I was mixing concerns (blackList of operations from `/data/` endpoint vs permissions for everything). Removed and will be properly reintroduced in next PR.